LayoutTests/imported/w3c/web-platform-tests/webcodecs/videoFrame-canvasImageSource-expected.txt

```diff
@@ -1,9 +1,9 @@
 
 
 FAIL <video> and VideoFrame constructed VideoFrame assert_unreached: Reached unreachable code
-FAIL CSSImageValue constructed VideoFrame assert_throws_dom: CSSImageValues are currently always tainted function "_ => { new VideoFrame(bgImage, {timestamp: 0}); }" threw object "ReferenceError: Can't find variable: VideoFrame" that is not a DOMException SecurityError: property "code" is equal to undefined, expected 18
-FAIL Image element constructed VideoFrame Can't find variable: VideoFrame
-FAIL SVGImageElement constructed VideoFrame Can't find variable: VideoFrame
-FAIL Canvas element constructed VideoFrame Can't find variable: VideoFrame
-FAIL Copy of canvas element constructed VideoFrame Can't find variable: VideoFrame
+FAIL CSSImageValue constructed VideoFrame assert_throws_dom: CSSImageValues are currently always tainted function "_ => { new VideoFrame(bgImage, {timestamp: 0}); }" threw object "TypeError: Member VideoFrameInit.visibleRect is required and must be an instance of DOMRectInit" that is not a DOMException SecurityError: property "code" is equal to undefined, expected 18
+FAIL Image element constructed VideoFrame Member VideoFrameInit.visibleRect is required and must be an instance of DOMRectInit
+FAIL SVGImageElement constructed VideoFrame Type error
+FAIL Canvas element constructed VideoFrame Member VideoFrameInit.visibleRect is required and must be an instance of DOMRectInit
+FAIL Copy of canvas element constructed VideoFrame Member VideoFrameInit.visibleRect is required and must be an instance of DOMRectInit
```

Comment:
Why are these failing, it looks like `VideoFrameInit.visibleRect` is defined correctly?

Reply:
I missed it, VideoFrameInit.visibleRect is not required, I'll fix it shortly